backend/app/helpers/spree/admin/stores_helper.rb

```diff
@@ -0,0 +1,19 @@
+module Spree
+  module Admin
+    module StoresHelper
+      def selected_checkout_zone(store)
+        return store.checkout_zone_id unless store.checkout_zone_id.nil?
+        return checkout_zone_id('No Limits') if Spree::Config[:checkout_zone].nil?
```

Comment:
@przemosk we should use translation or even better, add a `default` attribute to `Zone` model

Reply:
sure, will do that 👌🏻

Reply:
@damianlegawiec please take a look https://github.com/spree/spree/pull/10529/commits/0839df28402fe8dafdb54a414e575c98a80dcfcd

Reply:
@damianlegawiec by adding attribute you mean `attribute` to model by `attribute` or also add `default` with migration ?